backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -260,4 +260,16 @@ public String getCounty() {
   public PersonRole getRole() {
     return role;
   }
+
+  // these field names strings are used by Specification builders
+  public static class SpecField {
+    public static String PersonName = "nameInfo";
```

Comment:
However, these are constants and so should be `final` and be spelled PERSON_NAME and so forth.